.github/workflows/ReleaseIssue.yml

```diff
@@ -2,8 +2,9 @@
 name: Monthly Release Thread
 
 on:
-  schedule:
-   - cron: 0 0 1 * *  # Every 1st day of the month at 0000 – https://crontab.guru
+  # schedule:
+  #  - cron: 0 0 1 * *  # Every 1st day of the month at 0000 – https://crontab.guru
+  workflow_dispatch:
```

Comment:
This is the monthly release, so we need it on a schedule of the first of the month, so removing the cron job is not what we want.

Reply:
Ok, I'll set it back to scheduled. I was just trying to test out my changes first before submitting